config/jobs/kubernetes/sig-node/containerd.yaml

```diff
@@ -1338,4 +1338,32 @@ periodics:
   annotations:
     testgrid-dashboards: sig-node-kubelet
     testgrid-tab-name: kubelet-credential-provider
-    description: "tests alpha feature kubelet image credential provider"
+    description: "tests feature kubelet image credential provider"
+- name: ci-kubernetes-e2e-gcp-credential-provider
+  interval: 6h
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+  spec:
+    containers:
+      - args:
+        - --repo=k8s.io/kubernetes=master
+        - --timeout=240
+        - --root=/go/src
+        - --scenario=kubernetes_e2e
+        - --
+        - --check-leaked-resources
+        - --cluster=
+        - --env=ENABLE_AUTH_PROVIDER_GCP=true
+        - --env=KUBE_FEATURE_GATES=KubeletCredentialProviders=true
```

Comment:
if this is already beta (enabled by default), and about to go GA (locked on), let's not set the gate explicitly or we'll be creating debt we need to remove